### ISSUE TITLE
Seeking help

### ISSUE THREAD

**Issue by zhoudepei** [mathbaffa/Breast-IR-Imaging-Lateral-View-Segmentation-Maks#1]
Hello, dear blogger
Do you have details of the breast cancer patients in the DMR-IR-BC dataset that are labelled left or right breast?

**Comment by mathbaffa** [comment]
Hello @zhoudepei 
This dataset does not label each breast separately. You may find the dataset available in this link: http://visual.ic.uff.br/dmi/

**Comment by zhoudepei** [comment]
Thank you very much for writing back!
But I asked the administrator of http://visual.ic.uff.br/dmi/ for help some time ago and she told me that the annotation was done but on another dataset, and although I sent a help email to the holder of that dataset, I never got a reply.

**Comment by mathbaffa** [comment]
I'm really sorry but I don't know this other dataset.

**Comment by zhoudepei** [comment]
Hello, dear blogger
How can I download the paper "U-Net Convolutional Neural Networks for breast IR imaging segmentation on frontal and lateral view"? My school does not buy this paper and I have exhausted all the channels I know of and still can't download this paper.

**Comment by zhoudepei** [comment]
Hello, dear blogger
I am sorry to bother you again.
I was fortunate to find your paper on U-Net Convolutional Neural Networks for breast IR imaging segmentation on frontal and lateral view on a help site.Your paper is very well written, especially for training frontal and lateral images with The method of training different U-Net models for frontal and lateral images shows great flexibility. However, I have two questions that I need your help with:
1, In the paper you provided, I noticed that the mask data for the frontal image seems to be missing. Is it possible to provide this part of the data so that I can do relevant research?
2, I would also like to know how you processed the mask data for the side and front images. Would you be able to share the exact methodology used to process this data?
Looking forward to hearing back from you, thank you very much!

**Comment by mathbaffa** [comment]
Hello @zhoudepei
So regarding the frontal segmentations, you may find them at the Visual Labs website, check this link: http://visual.ic.uff.br/en/proeng/
The lateral segmentations were developed by my student and were checked by two radiologist specialists who confirmed the region of interest and the segmentation quality.
If you need assistance in finding the papers, just let me know and I can share a preprint version with you.

Best regards!

**Comment by zhoudepei** [comment]
Hello @mathbaffa 
Thank you very much for writing back. I have already obtained the paper through the assistance of enthusiastic users on the internet, for which I am truly grateful.

After carefully studying the link you provided (http://visual.ic.uff.br/en/proeng/) as well as the side masks labelled in your paper, I have two questions I would like to ask:

1、Why are the arms included in the breast position labelling in some of the original images? Is there a special consideration or need for such labelling?

2、What kind of software did you use for the labelling? I noticed that the curve of the breast edge is very smooth and precise. In contrast, I have had poor results with Labelme. Can you share the labelling tool you used and how you did it?

Thank you again for your reply and I look forward to hearing from you.

**Comment by mathbaffa** [comment]
@zhoudepei usually people study the arms as the primary site for breast cancer metastasis. So it's common to use the arms in the analysis. And I'm not sure but I think my student made the masks using photoshop.

**Comment by zhoudepei** [comment]
Hello @mathbaffa 
Thank you for writing back, and thank you so much for your patience in answering my questions.

**Comment by zhoudepei** [comment]
Hello @mathbaffa 
Hello, I have two questions I would like to ask:
1. Regarding the lateral view pictures in the DMR dataset used in your paper "U-Net Convolutional Neural Networks for breast IR imaging segmentation on frontal and lateral view", but you only used part of it (before No. 287), for No. Why did you not select the cases marked "Sick" after 287?
2. Why do many papers only mention the data before number 287? The paper states that "the DMR data set contains 287 cases", but there are 425 on the website http://visual.ic.uff.br/dmi/ Which one?
Sorry to bother you again and look forward to your reply!

**Comment by mathbaffa** [comment]
Hello @zhoudepei,
As the dataset is still collecting images, we now have more images compared to when we initially started this segmentation project. There isn't a specific reason for using only a portion of it. I strongly recommend that you make use of all the available images. This could be for the same reasons as other related projects.

**Comment by zhoudepei** [comment]
Hello @mathbaffa 
Thank you for writing back, and thank you so much for your patience in answering my questions.

**Comment by zhoudepei** [comment]
Hello @mathbaffa 
May I ask if the metrics mentioned in your published paper U-Net Convolutional Neural Networks for breast IR imaging segmentation on frontal and lateral view - accuracy refers to Is it the average accuracy of background and foreground or the recall of foreground (recall of foreground means the number of pixels predicted for foreground divided by the total number of pixels in foreground)?
Looking forward to your reply!
